demo-project/demo-api.raml

```diff
@@ -0,0 +1,88 @@
+#%RAML 1.0
+---
+title: Demo API
+baseUri: http://localhost:8080/
+
+/values:
+    get:
+      description: "Retrieve an array of value entries"
```

Comment:
Documentation is easiest to read if you either write complete sentences, or have an implied subject of _this URI_. In addition, I tend to write in more abstract terms (not mentioning encoding, format, etc.) unless really needed. The details are in the JSON schema and examples anyway.

The description is also lacking which values are returned.

 So, here I'd recommend something like _returns all values currently stored by the service_.